pulp_puppet_plugins/pulp_puppet/forge/views/releases.py

```diff
@@ -93,20 +96,61 @@ def _get_credentials(headers):
             return username, password
 
     @staticmethod
-    def _get_module_name(get_dict):
+    def _get_parameters(get_dict, path):
         """
-        :return: name of the module being requested, or None if not found or invalid
+        Get the parameters of the HTTP request
+
+        :param get_dict: The GET parameters
+        :type get_dict: dict
+        :param path: The path of the HTTP request
+        :type path: str
+        :return: dictionary with all of the request parameters,
+                 or HttpResponseBadRequest if invalid
+        :rtype: dict
         """
-        module_name = get_dict.get('module', '')
+        module_parameters = get_dict.copy()
+        module_name = module_parameters.get('module', '')
         match = MODULE_PATTERN.match(module_name)
         if match:
             normalized_name = u'%s/%s' % (match.group(1), match.group(3))
-            return normalized_name
+            module_parameters['module'] = normalized_name
+            return module_parameters
+        else:
+            return HttpResponseBadRequest('Module name is missing.')
 
 
 class ReleasesPost36View(ReleasesView):
 
     @staticmethod
+    def _get_parameters(get_dict, path):
+        """
+        Get the parameters of the HTTP request
+
+        :param get_dict: The GET parameters
+        :type get_dict: dict
+        :param path: The path of the HTTP request
+        :type path: str
+        :return: dictionary with all of the request parameters,
+                 or HttpResponseBadRequest if invalid
+        :rtype: dict
+        """
+        match = POST_36_URL_PATTERN.match(path)
+        if match:
+            path_parameter = match.group(1)
+            match = MODULE_PATH_PARAM_PATTERN.match(path_parameter)
```

Comment:
Consider not reusing `match` but instead keep them as two separate filenames. It makes the `if` logic confusing.